compiler/testData/ir/irText/classes/enum.txt

```diff
@@ -319,9 +319,6 @@ FILE fqName:<root> fileName:/enum.kt
               public final fun <get-ordinal> (): kotlin.Int declared in <root>.TestEnum4
             $this: VALUE_PARAMETER name:<this> type:kotlin.Enum<<root>.TestEnum4>
         PROPERTY FAKE_OVERRIDE name:x visibility:public modality:FINAL [val]
-          FIELD FAKE_OVERRIDE name:x type:kotlin.Int visibility:public [final]
```

Comment:
Is it expected that this fake field override was removed?

Reply:
Yes, the field we're overriding here is now private and as far as I can tell should not generate fake overrides. That's one of the properties that [`CheckIrElementVisitor`](https://github.com/JetBrains/kotlin/blob/e8ac22e238f3b108a11215292f92e39726691964/compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/CheckIrElementVisitor.kt#L307) tests for. Do you agree?

Reply:
Yes, absolutely. Thanks!